sass/_moj-overrides.scss

```diff
@@ -1,3 +1,35 @@
 .moj-pagination__results {
   margin: 0;
 }
+
+.moj-filter__selected {
```

Comment:
this now has to be white since now have the "hide filter" button in it, which itself is grey